code/modules/ruins/lavalandruin_code/sin_ruins.dm

```diff
@@ -115,7 +115,14 @@
 	for(var/level in levels)
 		if(!is_teleport_allowed(level))
 			levels -= level
-
+	if(user.z != 3) //if you somehow manage to bloody get out of lavaland without closing the UI
+		var/turf/naughty_naughty = locate(user.x, user.y, 3) //return to sender
```

Comment:
while naming variables funny things is humorous (and less of an issue in a local case), down the road if someone decides to touch this code it's going to be confusing at first. This was an issue I ran into when refactoring photocopier code because some decided to name the mob reference var to `ass` where it was then used throughout the entire 300 line dm file.

Reply:
This is what happens when I code while drunk 😆 I'll change it later